src/libraries/pkg/test/testPackages.proj

```diff
@@ -45,6 +45,9 @@
     <TestSupportFiles Include="$(LibrariesProjectRoot)shims\netfxreference.props">
       <DestinationFolder>$(TestToolsDir)</DestinationFolder>
     </TestSupportFiles>
+    <TestSupportFiles Include="$(RepoRoot)NuGet.config">
+      <DestinationFolder>$(TestSupportDir)</DestinationFolder>
```

Comment:
Are you expecting this to be picked up by the test project as well or just by test.msbuild? The former would not see this file in CI.

Reply:
There appears to be no directory in common with Helix payload structure and local structure.  If this is changed to `$(TestDir)` it produces the correct layout locally but that's not present in helix :/.  I guess we only need it to be present in Helix since that's the only place where it wouldn't find it in directory root. (since artifacts is below).

I'm debating just removing nuget.config entirely.  We actually don't need to hit any feeds except nuget.org.  The only place we need an additional feed is https://github.com/dotnet/runtime/blob/54fdd2606170f0b0584eea78c93ae554deaacdf5/src/libraries/pkg/test/packageSettings/System.IO.Ports/workaroundDowngrade.targets#L4 but that could be done differently, using an offically released package.

Thoughts?

Reply:
Why not just put the NuGet.config into a central location and point to it during the restore? There is a switch available for that. I'm changing the layout slightly with https://github.com/dotnet/runtime/pull/53439 so I would recommend to wait for that to go in (which hopefully will in 1-2h).

Reply:
Any thoughts on this NuGet.config being more than we need for package restore?  

> We actually don't need to hit any feeds except nuget.org

In fact, we probably don't want our packages depending on things from the internal feeds.

Reply:
> Any thoughts on this NuGet.config being more than we need for package restore in testing?

Nope I think that's fine. We follow the same approach in the installer tests in the runtime repo (they copy the NuGet.config over and add to it).

> In fact, we probably don't want our packages depending on things from the internal feeds.

That's right, but we don't reference any internal feeds from our NuGet.config.